zerver/lib/export.py

```diff
@@ -818,7 +824,7 @@ def export_from_config(
 
         assert model is not None
         try:
-            query = model.objects.filter(**filter_params)
+            query = model.objects.filter(**filter_params).order_by("id")
```

Comment:
Should this by `pk` instead of `id`? I think we might have a table or two that don't have an `id` field.